k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -510,8 +510,11 @@ function deployKafka()
         --values $TMP_KAFKA_ENVVARS_YAML  \
         --wait
 
-    rm $TMP_KAFKA_ENVVARS_YAML
```

Comment:
We want this line in there so we don't leave leftover ephemeral installation files. Unless there's some specific reason, I would add it back in.

Reply:
Thanks!  I can't believe I deleted this!  Will replace.